tests/mppnccombine/with-end-number

```diff
@@ -0,0 +1,208 @@
+data:
+  grid_xt = 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0 ;
+  grid_yt = 2.0, 3.0 ;
+  time = 0 ;
+  var =
+    20, 21, 22, 23, 24, 25, 26, 27, 28, 29,
+    30, 31, 32, 33, 34, 35, 36, 37, 38, 39 ;
+}
+EOF
+
+ncgen -o decomp.nc.0005 << EOF || framework_failure_
+netcdf compress {
+dimensions:
+  grid_xt = 10 ;
+  grid_yt = 2 ;
+  time = UNLIMITED ; // (1 currently)
+variables:
+  double grid_xt(grid_xt) ;
+    grid_xt:long_name = "T-cell longitude" ;
+    grid_xt:units = "degrees_E" ;
+    grid_xt:axis = "X" ;
+    grid_xt:domain_decomposition = 1, 10, 1, 10 ;
+  double grid_yt(grid_yt) ;
+    grid_yt:long_name = "T-cell latitude" ;
+    grid_yt:units = "degrees_N" ;
+    grid_yt:axis = "Y" ;
+    grid_yt:domain_decomposition = 1, 8, 5, 6 ;
+  double time(time) ;
+    time:long_name = "time" ;
+    time:units = "days since 0001-01-01 00:00:00" ;
+    time:axis = "T" ;
+    time:calendar_type = "NOLEAP" ;
+    time:calendar = "noleap" ;
+  float var(grid_yt, grid_xt) ;
+    var:long_name = "Decomposed variable" ;
+    var:units = "m" ;
+    var:missing_value = 1.e+20f ;
+    var:_FillValue = 1.e+20f ;
+    var:dummy_attribute = "just a dummy attribute" ;
+// global attributed:
+  :filename = "decomp.nc.0005" ;
+  :NumFilesInSet = 4 ;
+  :dummy_global_att = "a global dummy attribute" ;
+data:
+  grid_xt = 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0 ;
+  grid_yt = 4.0, 5.0 ;
+  time = 0 ;
+  var =
+    40, 41, 42, 43, 44, 45, 46, 47, 48, 49,
+    50, 51, 52, 53, 54, 55, 56, 57, 58, 59 ;
+}
+EOF
+
+ncgen -o decomp.nc.0006 << EOF || framework_failure_
+netcdf compress {
+dimensions:
+  grid_xt = 10 ;
+  grid_yt = 2 ;
+  time = UNLIMITED ; // (1 currently)
+variables:
+  double grid_xt(grid_xt) ;
+    grid_xt:long_name = "T-cell longitude" ;
+    grid_xt:units = "degrees_E" ;
+    grid_xt:axis = "X" ;
+    grid_xt:domain_decomposition = 1, 10, 1, 10 ;
+  double grid_yt(grid_yt) ;
+    grid_yt:long_name = "T-cell latitude" ;
+    grid_yt:units = "degrees_N" ;
+    grid_yt:axis = "Y" ;
+    grid_yt:domain_decomposition = 1, 8, 7, 8 ;
+  double time(time) ;
+    time:long_name = "time" ;
+    time:units = "days since 0001-01-01 00:00:00" ;
+    time:axis = "T" ;
+    time:calendar_type = "NOLEAP" ;
+    time:calendar = "noleap" ;
+  float var(grid_yt, grid_xt) ;
+    var:long_name = "Decomposed variable" ;
+    var:units = "m" ;
+    var:missing_value = 1.e+20f ;
+    var:_FillValue = 1.e+20f ;
+    var:dummy_attribute = "just a dummy attribute" ;
+// global attributed:
+  :filename = "decomp.nc.0006" ;
+  :NumFilesInSet = 4 ;
+  :dummy_global_att = "a global dummy attribute" ;
+data:
+  grid_xt = 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0 ;
+  grid_yt = 6.0, 7.0 ;
+  time = 0 ;
+  var =
+    60, 61, 62, 63, 64, 65, 66, 67, 68, 69,
+    70, 71, 72, 73, 74, 75, 76, 77, 78, 79 ;
+}
+EOF
+
+# run mppnccombine to generate decomp.nc, not
+# listing other input files
+mppnccombine -f -e 6 decomp.nc || skip_
```

Comment:
So with the new testing framework, is this command run and then marked as a skip if it fails?

Reply:
This test is not working, as I couldn't figure out what the reason for this option is and how to get it to work.  Thus, if it starts to work -- great.  If not, I skip it as I don't expect it to work.  This is the only test like this.